lib/matplotlib/patches.py

```diff
@@ -1394,8 +1395,9 @@ class Ellipse(Patch):
     A scale-free ellipse.
     """
     def __str__(self):
-        return "Ellipse(%s,%s;%sx%s)" % (self.center[0], self.center[1],
-                                         self.width, self.height)
+        pars = self.center[0], self.center[1], self.width, self.height, self.angle
```

Comment:
I've amended the last commit to wrap to 80 char here.
And I've thrown in a wedge, arc and test docstring.
Hope you like it.